src/PKSim.Presentation/Presenters/Main/MenuAndToolBarPresenter.cs

```diff
@@ -265,17 +276,18 @@ private void disableSimulationItems()
          updateSimulationItemsAccordingToSimulationState();
       }
 
-      private void updateSimulationItemsAccordingToSimulationState()
+      private void updateSimulationItemsAccordingToSimulationState(bool isRunning = false)
```

Comment:
Same question as in another PR. Why are we adding parameters in private methods with default values. Can you explain the reasoning because I am not getting it.

Reply:
It was my suggestion; my justification is that I'm lazy. But that's not a good justification ;)